ui/src/app/components/load-dump/load-dump.component.html

```diff
@@ -25,7 +25,7 @@ <h3>Dump File</h3>
         />
       </mat-form-field>
       <br />
-      <button mat-raised-button type="submit" color="primary" [disabled]="!connectForm.valid">
+      <button mat-raised-button type="submit"  [disabled]="!connectForm.valid">
```

Comment:
Better not remove **color="primary"**. Instead add **.mat-raised-button.mat-primary {
  color: white;
  background-color: #3367d6;
}** in global. This will also us to trigger mat-raised button with primary color.

Reply:
Made the suggested changes